src/components/Search/Search.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import { MoviesList } from '../MoviesList';
+
+export class Search extends Component {
+  state = {
+    query: '',
+  };
+
+  handleChange = (event) => {
+    const { name, value } = event.target;
+
+    this.setState({ [name]: value });
+  };
```

Comment:
```suggestion
  handleChange = (event) => {
    this.setState({ query: event.target.value });
  };
```
Well, it may be done in a more simple way. But ok